public/sass/screen.scss

```diff
@@ -4,7 +4,6 @@
 
```

Comment:
this diff seems super noisy, did you actually change anything in this file? if not i'd revert.